doc/ref_cert/lfn/chapters/chapter02.md

```diff
@@ -5,15 +5,48 @@
 
 ## Table of Contents
 * [2.1 Introduction](#2.1)
-* [2.2 Heading](#2.2)
-* [2.3 Heading](#2.3)
+* [2.2 Methodology](#2.2)
+* [2.3 Certification Strategy & Vehicle](#2.3)
+* [2.4 Profiles Reference](#2.4)
+* [2.5 Compliance, Verification, and Certification](#2.5)
+* [2.6 Entry & Exit Criteria](#2.6)
+* [2.7 Frameworks](#2.7)
 
 <a name="2.1"></a>
 ## 2.1 Introduction
 
-<a name="1.2"></a>
-## 2.2 Heading
+Describe the purpose.. 
 
+- Define E2E Framework Requirements, process and automation for NFVI Certification
+- Ensure Platform Stability, Operability, & Performance for target RAs
 
-<a name="1.3"></a>
-## 2.3 Heading
+<a name="2.2"></a>
+## 2.2 Methodology
+
+Define the approach to NFVI testing, and Frameworks needed...
+
+<a name="2.3"></a>
+## 2.3 Certification Strategy & Vehicle
+
+Define and illustrate the certification strategy and vehicle for NFVI..
+
+<a name="2.4"></a>
+## 2.4 Profiles Reference 
+
+Define the profiles used, why they're important, and what is being certified..
+
+<a name="2.5"></a>
+## 2.5 Compliance, Verification, and Certification
+
+Define the framework and process for NFVI certification..
+
+<a name="2.6"></a>
+## 2.6 Entry & Exit Criteria
+
+Provide a list of entry and exit criteria to be used for NFVI certifications..
+
+<a name="2.7"></a>
+## 2.7 Frameworks
+
+List out the various frameworks used in certifications
+- e.g. Functest, StorePerf, SampleVNF, etc)
```

Comment:
@mf4716 I wold envisioned this not to specify specific project and focus on requirements only.